CaroteneSkills.md

```diff
@@ -41,26 +41,43 @@ Currently only Carotene v3 is supported. See the documentation for [Job Title se
 for more information about carotene classification and carotene IDs.
 
 ## Response Structure
-The response consist of a list of `skills` where each item contains a `term` string .
+Response consist of a `carotene_id`, `carotene_title`, and list of `skills` for the requested 
+`carotene_id`. Each item in the `skills` list contains `term`, `id` strings, and `description` 
```

Comment:
do we have an extra comma? strings, and

Reply:
Hmm not an extra comma but one too many occurrences of 'strings'. Fixed that part. Should be 

> Each item in the `skills` list contains `term`, `id`, and `description` 
strings as well as a list of links to relevant videos.

... or, I suppose 

> Each item in the `skills` list contains `term`, `id` and `description` 
strings as well as a list of links to relevant videos.

BTW, this is seriously one of the best wikipedia pages: https://en.wikipedia.org/wiki/oxford_comma 

> The Times once published an unintentionally humorous description of a Peter Ustinov documentary, noting that "highlights of his global tour include encounters with Nelson Mandela, an 800-year-old demigod and a dildo collector". This would still be ambiguous if a serial comma were added, as Mandela could still be mistaken for a demigod, although he would be precluded from being a dildo collector.

[Who gives a fuck about an Oxford comma?](https://www.youtube.com/watch?v=P_i1xk07o4g)